include/fcl/geometry/shape/convex.h

```diff
@@ -51,7 +51,37 @@ class FCL_EXPORT Convex : public ShapeBase<S_>
 
   using S = S_;
 
-  /// @brief Constructing a convex, providing normal and offset of each polytype surface, and the points and shape topology information 
+  // TODO(SeanCurtis-TRI): Figure out what the ownership of this data is. It
+  // *seems* that this struct only owns the edges.
+  /// @brief Definition of a convex polytope.
+  ///
+  /// This requires *two* coordinated representations of the convex hull:
+  ///   - The set of half spaces that bound the hull region and
+  ///   - the polytope mesh formed by the intersection of the half spaces.
+  ///
+  /// The half spaces are defined by the implicit equation of a plane:
+  /// `Π(x, y, z) = Ax + By + Cz + d = 0` where `n̂ = [A, B, C]` and |n̂| = 1.
+  /// The normal points *outside* of the convex region. There should be *no*
+  /// redundant planes in the set.
+  ///
+  /// For each plane Π there should corresponding face f that lies on that plane
```

Comment:
This was one thing I was concerned about. It is possible to have several triangles in the convex mesh that lie on a single plane. So based on this description a user must loop through a mesh and combine all triangle that are on a given plane in to a single polytope. Do I understand this correctly?